scripts/hud/cgaz.js

```diff
@@ -88,17 +88,7 @@ class Cgaz {
 	static bShouldUpdateStyles = false;
 
 	static onLoad() {
-		if (GAMEMODE_WITH_NULL[GameModeAPI.GetCurrentGameMode()].name === 'Defrag') {
-			this.updateEventHandle ??= $.RegisterEventHandler(
-				'ChaosHudProcessInput',
-				$.GetContextPanel(),
-				this.onUpdate.bind(this)
-			);
-		} else {
-			this.velocityArrow.visible = false;
-			this.windicatorArrow.visible = false;
-			return;
-		}
+		if (GAMEMODE_WITH_NULL[GameModeAPI.GetCurrentGameMode()].name !== 'Defrag') return;
```

Comment:
This is really weird. We should define an enum object that follows the C++ style of gamemodes, so you can do `GameModeAPI.GetCurrentGameMode() !== GameMode.DEFRAG` instead. This can be done in another PR, so this is just a nit for now.

Reply:
Created an issue for it at #23